### PR TITLE
Fix(GraphQL): Add header info for subscriptions

### PR DESCRIPTION
**Pull request by JatinDev543**
This PR adds header info for subscriptions.

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (pawanrawal) [inline .idea/.gitignore]: remove these files which are under `.idea` folder.

File: graphqlws/internal/connection/connection.go [hunk 2]
```diff
@@ -165,7 +166,7 @@ func (conn *connection) readLoop(ctx context.Context, send sendFunc) {
 				continue
 			}
 			send("", typeConnectionAck, nil)
-                       
+			header =msg.Payload
```
Comment (pawanrawal) [inline graphqlws/internal/connection/connection.go]: Run `go fmt` on this. It is not formatted properly.

**Review by pawanrawal — APPROVED** [review]
LGTM, but please address the comments before merging.

File: .idea/.gitignore (hunk 1 above)
Comment (JatinDev543) [reply .idea/.gitignore]: > remove these files which are under `.idea` folder.

removed

File: graphqlws/internal/connection/connection.go (hunk 2 above)
Comment (JatinDev543) [reply graphqlws/internal/connection/connection.go]: formatted properly.

**Pull request merged**